client/src/app/pages/controls/stakeholder-groups/stakeholder-groups.tsx

```diff
@@ -253,71 +257,71 @@ export const StakeholderGroups: React.FC = () => {
               }
               numRenderedColumns={numRenderedColumns}
             >
-              {currentPageItems?.map((stakeholderGroup, rowIndex) => {
-                return (
-                  <Tbody
-                    key={stakeholderGroup.id}
-                    isExpanded={isCellExpanded(stakeholderGroup)}
-                  >
-                    <Tr {...getTrProps({ item: stakeholderGroup })}>
-                      <TableRowContentWithControls
-                        {...tableControls}
-                        item={stakeholderGroup}
-                        rowIndex={rowIndex}
-                      >
-                        <Td width={25} {...getTdProps({ columnKey: "name" })}>
-                          {stakeholderGroup.name}
-                        </Td>
-                        <Td
-                          width={10}
-                          {...getTdProps({ columnKey: "description" })}
+              <Tbody key={stakeholderGroups?.length}>
```

Comment:
I know multiple `Tbody` tags are allowable, but with PF styling things could get wonky.  Good idea to pull it out!